contracts/liquidity_hub/fee_collector/src/commands.rs

```diff
@@ -289,6 +289,7 @@ pub fn aggregate_fees(
                             operations,
                             minimum_receive: None,
                             to: None,
+                            max_spread: Some(Decimal::percent(50u64)),
```

Comment:
Add the max_spread possible when doing the fees aggregation to avoid max spread assertions